php/README.md

```diff
@@ -0,0 +1,25 @@
+# Custom Auth with php
+
+The core logic is in `Ionic\CustomAuthenication` class which can be subclassed
```

Comment:
Haha, there's no reason examples can't be abstracted into small functions. But you have to admit an imperative style is useful for something as complex as this. 😄